### PR TITLE
Update and rename login web to login web.html

### PR DESCRIPTION
**Pull request by HeraclitusHHX**
add "hide" and "show"

### CODE REVIEW

File: login web.html [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+<head>
+     <title>WebServer Login</title>
+</head>
+<h1>Webserver Login</h1>
+<body>
+<form id="login" action="/login" method="post">
+   <div class="two-thirds column" id="main">
+     <fieldset>
+      <legend>Login</legend>
+      <input type="radio" id="guestradio" name="heard" value="guest">
```
Comment (jiaju-yang) [inline login web.html:12]: 不要连写单词，用underline naming 或 upper camel case

**Comment by HeraclitusHHX** [comment]
`<script>
var xhr = new XMLHttpRequest
    xhr.onload = function(){
    if (xhr.status===200)
       var ccc = xhr.responseText
       $('h1').after('<p>'+ccc+'</p>')
    }
xhr.open('GET','https://service.api.psychesworld.com/auth/tip','ture')
xhr.send(null)
var script = document.createElement('script');  
script.src = "http://aa.xx.com/js/*.js";  
document.div.appendChild(script);`
</script>

**Comment by jiaju-yang** [comment]
代码呢？

**Comment by jiaju-yang** [comment]
不要显示整个json给用户，json里面的键是供你取出数据，值才是显示的

**Comment by HeraclitusHHX** [comment]
是所有代码吗？？我已经改到dev里了

**Comment by HeraclitusHHX** [comment]
中午点改了哈

**Comment by HeraclitusHHX** [comment]
<script>
var xhr = new XMLHttpRequest
    xhr.onload = function(){
    if (xhr.status===200)
      
 var ccc = xhr.responseText
       var responseObject=JSON.parse(ccc)
       $('h1').after('<p>'+responseObject.tip+'</p>')
    }
xhr.open('GET','https://service.api.psychesworld.com/auth/tip','ture')
xhr.send(null)
var script = document.createElement('script');  
script.src = "http://aa.xx.com/js/*.js";  
document.div.appendChild(script);
</script>

**Pull request closed without merging**